metron-deployment/packaging/ambari/metron-mpack/src/main/resources/common-services/METRON/CURRENT/package/scripts/metron_service.py

```diff
@@ -70,6 +70,11 @@ def build_global_config_patch(params, patch_file):
         "path": "/es.date.format",
         "value": "{{es_date_format}}"
     },
+    {
+        "op": "add",
+        "path": "/es.xpack.user",
+        "value": "{{es_xpack_user}}"
```

Comment:
We can't store passwords like this in plain text because they'll end up in Zookeeper as plaintext without permissions locking it down.

Reply:
Good point. Is there an existing encryption mechanism that we can use to prevent plain text passwords here?

Reply:
could we lean on the hadoop credentials apis?